extensions/3DTILES_metadata/schema/3DTILES_metadata.schema.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://json-schema.org/draft-04/schema",
+  "title": "3DTILES_metadata object",
+  "type": "object",
+  "description": "An object containing description of metadata available",
+  "properties": {
+    "tileset": {
+      "$ref": "3DTILES_metadata.tileset.schema.json"
+    },
+    "tile": {
+      "$ref": "3DTILES_metadata.tile.schema.json"
+    },
+    "content": {
+      "$ref": "3DTILES_metadata.content.schema.json"
+    },
+    "featureClass": {
```

Comment:
I realize this is going to move around a bit, but note that it should probably be `featureClasses` plural.

Reply:
Now I'm not quite sure, see @lilleyse's comment here: https://github.com/CesiumGS/glTF/pull/3#discussion_r503616409